<ui version="4.0" >
 <class>NewConnectionDlgClass</class>
 <widget class="QWidget" name="NewConnectionDlgClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>460</width>
    <height>354</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>New Connection</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="RoseApple.qrc" >
    <normaloff>:/RoseApple/Resources/new.png</normaloff>:/RoseApple/Resources/new.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout_3" >
   <property name="sizeConstraint" >
    <enum>QLayout::SetFixedSize</enum>
   </property>
   <property name="margin" >
    <number>4</number>
   </property>
   <property name="spacing" >
    <number>4</number>
   </property>
   <item row="0" column="0" >
    <layout class="QGridLayout" name="gridLayout_2" >
     <item row="0" column="0" >
      <layout class="QHBoxLayout" name="horizontalLayout_3" >
       <item>
        <widget class="QLabel" name="label" >
         <property name="minimumSize" >
          <size>
           <width>62</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize" >
          <size>
           <width>62</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text" >
          <string>Driver:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="driveCmbbox" >
         <property name="editable" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0" >
      <widget class="QWidget" native="1" name="editWidget" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <layout class="QFormLayout" name="formLayout" >
        <property name="horizontalSpacing" >
         <number>4</number>
        </property>
        <property name="verticalSpacing" >
         <number>4</number>
        </property>
        <property name="margin" >
         <number>4</number>
        </property>
        <item row="0" column="0" >
         <widget class="QLabel" name="hostLbl" >
          <property name="minimumSize" >
           <size>
            <width>62</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize" >
           <size>
            <width>62</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text" >
           <string>Host:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1" >
         <widget class="QLineEdit" name="hostEdit" />
        </item>
        <item row="1" column="0" >
         <widget class="QLabel" name="label_6" >
          <property name="minimumSize" >
           <size>
            <width>62</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize" >
           <size>
            <width>62</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text" >
           <string>DB Name:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1" >
         <widget class="QLineEdit" name="nameEdit" />
        </item>
        <item row="2" column="0" >
         <widget class="QLabel" name="label_3" >
          <property name="minimumSize" >
           <size>
            <width>62</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize" >
           <size>
            <width>62</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text" >
           <string>Port:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1" >
         <widget class="QLineEdit" name="portEdit" />
        </item>
        <item row="3" column="0" >
         <widget class="QLabel" name="userLbl" >
          <property name="minimumSize" >
           <size>
            <width>62</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize" >
           <size>
            <width>62</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text" >
           <string>User:</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1" >
         <widget class="QLineEdit" name="userEdit" />
        </item>
        <item row="4" column="0" >
         <widget class="QLabel" name="passwordLbl" >
          <property name="minimumSize" >
           <size>
            <width>62</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize" >
           <size>
            <width>62</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text" >
           <string>Password:</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1" >
         <layout class="QHBoxLayout" name="horizontalLayout_4" >
          <item>
           <widget class="QLineEdit" name="passwordEdit" >
            <property name="echoMode" >
             <enum>QLineEdit::Password</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="savePwdChkbox" >
            <property name="text" >
             <string>Save</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item row="2" column="0" >
      <widget class="QWidget" native="1" name="fileWidget" >
       <layout class="QHBoxLayout" name="horizontalLayout_2" >
        <property name="margin" >
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="fileLbl" >
          <property name="minimumSize" >
           <size>
            <width>62</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize" >
           <size>
            <width>62</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text" >
           <string>File:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="fileEdit" >
          <property name="echoMode" >
           <enum>QLineEdit::Normal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="fileBtn" >
          <property name="text" >
           <string>...</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="1" >
    <layout class="QVBoxLayout" name="verticalLayout" >
     <item>
      <widget class="QPushButton" name="connectBtn" >
       <property name="text" >
        <string>&amp;Connect</string>
       </property>
       <property name="icon" >
        <iconset resource="RoseApple.qrc" >
         <normaloff>:/RoseApple/Resources/connect.png</normaloff>:/RoseApple/Resources/connect.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelBtn" >
       <property name="text" >
        <string>C&amp;ancel</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="btnSpacer" >
       <property name="orientation" >
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="connectionsBtn" >
       <property name="text" >
        <string>Co&amp;nnections</string>
       </property>
       <property name="checkable" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0" colspan="2" >
    <widget class="QWidget" native="1" name="extWidget" >
     <layout class="QGridLayout" name="gridLayout" >
      <property name="leftMargin" >
       <number>0</number>
      </property>
      <property name="topMargin" >
       <number>4</number>
      </property>
      <property name="rightMargin" >
       <number>0</number>
      </property>
      <property name="bottomMargin" >
       <number>2</number>
      </property>
      <property name="spacing" >
       <number>4</number>
      </property>
      <item row="0" column="0" >
       <widget class="QTableView" name="tableView" >
        <property name="contextMenuPolicy" >
         <enum>Qt::CustomContextMenu</enum>
        </property>
        <property name="selectionMode" >
         <enum>QAbstractItemView::SingleSelection</enum>
        </property>
        <property name="selectionBehavior" >
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <resources>
  <include location="RoseApple.qrc" />
 </resources>
 <connections/>
</ui>
